dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -67,25 +66,54 @@ def openFile(self,region,bucket,filename,csvSchema,bucketName,errorFilename):
             raise ResponseException("Error in header row: CSV file must use only '|' or ',' as the delimiter", StatusCode.CLIENT_ERROR, ValueError, ValidationError.headerError)
 
         self.delimiter = "|" if line.count("|") != 0 else ","
-        for row in csv.reader([line],dialect='excel', delimiter=self.delimiter):
-            for cell in row :
-                headerValue = FieldCleaner.cleanString(cell)
-                if( not headerValue in possibleFields) :
+
+        # check to see if header contains long or short column names
+        # TODO: is there a better way to do this?
+        validation_db = ValidatorValidationInterface()
+        longNameDict = validation_db.getLongToShortColname()
+        headerNames = line.split(self.delimiter)
```

Comment:
@jworcestBAH Thanks for these notes--I'm definitely in the market for a better way to determine for sure whether a submitted file is using the long or short column headers. I'm not sure that you mean, though, about "going around the csv package." Can you say more about that?